modules/core/src/main/java/org/apache/ignite/internal/processors/cache/IgniteCacheOffheapManagerImpl.java

```diff
@@ -3102,6 +3111,10 @@ public void restoreState(long size, long updCntr, @Nullable Map<Integer, Long> c
                 return oldRow;
             }
 
+            @Override public boolean oldRowExpiredFlag() {
```

Comment:
```suggestion
            /**
            * Flag that indicates if oldRow was expired during invoke.
            * @return {@code true} if old row was expired, {@code false} otherwise.
            */
            @Override public boolean oldRowExpiredFlag() {
```